packages/dds/tree/src/test/feature-libraries/default-schema/schemaChecker.spec.ts

```diff
@@ -625,24 +642,28 @@ describe.only("schema validation", () => {
 
 			// Confirm that the field supports number nodes
 			assert.equal(
-				isFieldInSchema([numberNode], fieldSchema, schemaCollection, schemaPolicy),
+				isFieldInSchema([numberNode], fieldSchema, {
+					schema: schemaCollection,
+					policy: schemaPolicy,
```

Comment:
if you made a single SchemaAndPolicy above instead of two separate constants, this and below would be simpler.